proto/storage/image_integration.proto

```diff
@@ -152,4 +152,6 @@ enum ImageIntegrationCategory {
   // so the image integration is also currently used for node integrations.
   // This decision was made because we currently only support one node scanner (our scanner).
   NODE_SCANNER = 2;
+  // TODO: ROX-32218: ensure scanner V4 can also be used as virtual machine scanner
+  VIRTUAL_MACHINE_SCANNER = 3;
```

Comment:
We would need to make sure that ROX-32218 is addressed before merging this as otherwise we would loose the ability to scan with current scanner V4.

Reply:
This change effectively makes VM scanning depend on a category that (per PR description) can only be set via API right now. 
VM scanning is expected to work without that today so we may need to add some compatibility fallback for that case (or a feature-flag maybe?).